src/networks/devnet/mod.rs

```diff
@@ -140,14 +140,20 @@ pub static HEIGHT_INFOS: Lazy<[HeightInfo; 22]> = Lazy::new(|| {
     ]
 });
 
-pub(super) static DRAND_SCHEDULE: [DrandPoint<'static>; 2] = [
-    DrandPoint {
-        height: 0,
-        config: &DRAND_MAINNET,
-    },
-    DrandPoint {
-        // height is TBD
-        height: i64::MAX,
-        config: &DRAND_QUICKNET,
-    },
-];
+pub(super) static DRAND_SCHEDULE: Lazy<[DrandPoint<'static>; 2]> = Lazy::new(|| {
+    [
+        DrandPoint {
+            height: 0,
+            config: &DRAND_MAINNET,
+        },
+        DrandPoint {
+            // height is TBD.
+            // likely to be `get_upgrade_epoch_by_height(HEIGHT_INFOS.iter(), Height::Pineapple).unwrap()`.
+            height: i64::MAX.max(
+                get_upgrade_epoch_by_height(HEIGHT_INFOS.iter(), Height::Watermelon)
+                    .unwrap_or_default(),
+            ),
```

Comment:
Hmm, how about we mark the function with `#[allow(dead_code)]` instead and explain that the function will be used in the future in a comment?

Reply:
Fixed